tools/faucet/cmd/cli.go

```diff
@@ -32,6 +32,10 @@ const (
 	serverPortName    = "serverPort"
 	serverPortDefault = 80
 	serverPortUsage   = "Port where the web server binds to"
+
+	defaultAmountName    = "defaultAmount"
+	defaultAmountDefault = 100.0
+	defaultAmountUsage   = "Default amount of token to fund (in ETH)"
```

Comment:
it's a pet annoyance, I'll leave it to you consideration. The flags receive ETH but the config struct receives WEI. Worth normalizing one way or the other imo.

Reply:
I feel like we should mostly use explicit big.Int for token values everywhere in code, seems to be mostly what geth does.

But the CLI is a human configured interface, imo it makes sense for it to use human readable values but not to leak that into other layers. If I'm checking the config for each env I don't want to have to count the zeroes ideally.